libraries/AP_MSP/AP_MSP.cpp

```diff
@@ -44,7 +44,7 @@ const AP_Param::GroupInfo AP_MSP::var_info[] = {
     // @Description: A bitmask to set some MSP specific options
     // @Bitmask: 0:EnableTelemetryMode, 1: DJIWorkarounds
     // @User: Standard
-    AP_GROUPINFO("_OPTIONS", 2, AP_MSP, _options, OPTION_TELEMETRY_DJI_WORKAROUNDS),
+    AP_GROUPINFO("_OPTIONS", 2, AP_MSP, _options, (uint8_t)Option::TELEMETRY_DJI_WORKAROUNDS),
```

Comment:
Hi, this is not my code, but yes, it will be set as default for everyone but is actually honored only by the DJI backend.
It went in with this [PR](https://github.com/ArduPilot/ardupilot/pull/17569)

Reply:
@shellixyz any comments on this?

Reply:
Seems good to me. To be honest we can probably remove this option and force the "workaround". It only impacts the MSP telemetry when it is used through the DJI OSD driver anyway and I don't know any DJI device which is supposed to use the "non-workaround" format. Or we can switch it to a `TELEMETRY_DISABLE_DJI_WORKAROUNDS` option bit, in case a future version can display data for more than one motor (unlikely to happen in my opinion), it doesn't bother me.

Reply:
@shellixyz @peterbarker let's switch to TELEMETRY_DISABLE_DJI_WORKAROUNDS in case a future version can display data for multiple motors (very unlikely)